help-docs/marketplace/apps/proof-of-humanity.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 16
+title: Proof of Humanity
+id: rarimo-poh
+sidebar_label: Proof of Humanity
+---
+
+Keep your Discord server bot-free using Rarimo’s Proof-of-Humanity miniapp.
+
+### About this app
+
+Rarimo’s Proof-of-Humanity miniapp ensures that only verified humans are present either within a server or channel. It keeps the community server spam and scam-free, which saves mods’ time, improves the quality of conversation, and helps protect airdrops.
+
+Members can select between five identity providers to verify their humanity and will receive a verified human credential in their [RariMe extension for MetaMask](https://rarime.com/) that they can use across Web3 for perks and exclusive access. This credential will be used to generate a zero-knowledge proof that permits minting a [Proof of Humanity SBT on Polygon](https://opensea.io/collection/proof-of-humanity-2). This SBT is then used for token gating.
+
+Learn more about [Proof of Humanity](https://docs.rarimo.com/use-cases/proof-of-humanity) by Rarimo.
+
+### Getting started
+
+To get started, the server Admin should follow these steps:
+
+1. Install the Proof of Humanity Miniapp from the [Collab.Land Marketplace](https://cc.collab.land/).
+1. Create a Discord role for verified humans.
+1. Create the following [token-gating rule](/help-docs/key-features/token-gate-communities) in the Command Center:

```

Comment:
in a later app version, would be great if the miniapp did this TGR creation for the admins. is that possible @raymondfeng?